addon/chrome/skin/app_version/78/win/tabStyles.css

```diff
@@ -22,7 +22,7 @@
   @media not all and (-moz-os-version: windows-vista) {
     @media not all and (-moz-os-version: windows-win7) {
       @media not all and (-moz-os-version: windows-win8) {
-        #tabbrowser-tabs[tabmix_currentStyle~="bg"] > .tabbrowser-tab[visuallyselected="true"] > .tab-stack > .tab-background > .tab-background-middle {
+        #tabbrowser-tabs[tabmix_currentStyle~="bg"]  .tabbrowser-tab[visuallyselected="true"] > .tab-stack > .tab-background > .tab-background-middle {
```

Comment:
Why did you remove > between ```#tabbrowser-tabs``` and ```.tabbrowser-tabs```
```#tabbrowser-tabs[tabmix_currentStyle~="bg"] > .tabbrowser-tab```

Doesn't  tabbrowser-tab direct child of tabbrowser-tabs? 
It is better for specificity with parent > child.
in some places you replace .tabbrowser-tab class with id #tabbrowser-tab.
I think we should be consistent

This comment apply to all other changes.

Reply:
Not any more. Now there is other things between  `#tabbrowser-tabs` and `.tabbrowser-tabs`

I only replace things like `.tabbrowser-arrowscrollbox` to `#tabbrowser-arrowscrollbox` this basically matching changes of firefox.

Reply:
OK, bet lets follow Firefox convention:
replace ```#tabbrowser-tabs[tabmix_currentStyle~="bg"] > .tabbrowser-tab[visuallyselected="true"]```
with ```#tabbrowser-tabs[tabmix_currentStyle~="bg"] > #tabbrowser-arrowscrollbox > .tabbrowser-tab[visuallyselected="true"]```

Reply:
This have been done.

Reply:
This have been done.